apps/explorer/src/pages/validator/ValidatorDetails.tsx

```diff
@@ -21,6 +22,14 @@ function ValidatorDetails() {
         return data.activeValidators.find((av) => av.suiAddress === id) || null;
     }, [id, data]);
 
+    const atRiskRemainingEpochs = useMemo(() => {
```

Comment:
nit: does this need to be memo'd for perf reasons? I'd recommend against it unless the operation is expensive and there's a tangible benefit to memo'izing this value

https://react.dev/reference/react/useMemo#how-to-tell-if-a-calculation-is-expensive

Reply:
This is more to contain the whole thing inside `useMemo` rather than for the memoized purpose. But I can break it out into a different function to do the same thing.